_posts/release/2022-11-02-accumulo-2.1.1.md

```diff
@@ -42,6 +42,7 @@ Apache Accumulo 2.1.1 is a patch release of the 2.1.0 LTM line.
 * {% ghi 3023 %}, {% ghi 3053 %} Prevent excessive logging when an ExternalCompaction
   is cancelled for cause and around the ThreadPools
 * {% ghi 3057 %} Modified Monitor to direct DataTable errors to console
+* {% ghi 3205 %} Support writing cluster config parser output to a file
```

Comment:
This is an internal tool... not really user facing. I'm not sure it's important enough to highlight in the release notes. But, the whole release notes will be reviewed before release... so it doesn't matter. Go ahead and merge.

Reply:
Fair enough, I figured I'd add it just in case since it was a draft. As you said, can remove later.

Reply:
Yeah, but I guess my point is that because it's draft, you don't really need a code review for such trivial updates to the draft.